tools-v2/internal/utils/base.go

```diff
@@ -0,0 +1,266 @@
+package cobrautil
```

Comment:
Is it the same as pkg/cli/command/base.go?

Reply:
If it is to solve the circular reference, you can create another package called copyset under the internal, so that the circular reference can be avoided?

Reply:
It is not elegant to maintain two senses